pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools", "wheel", "cython"]  # PEP 508 specification
```

Comment:
It's tempting to do the PEP 508 stuff in a separate PR, just to give it a little more visibility. This looks harmless, but it might be good to seed discussion about any further PEP508 changes